### DISCUSSION TITLE
Matching boundary conditions between two domains

### DISCUSSION THREAD

**Discussion by RomanB22** [zwicker-group/py-pde#380] [Q&A]
Hi, thank you so much for such an amazing package!

I apologize if the question is very basic, I'm still new with the use of this package. I'm trying to solve a Fokker-Planck equation in 2+1 D of the form $\partial_t P(x,y,t) = \frac{D}{2} \partial_x^2 P - f1(x,y,t) \partial_x P + f2(x,y) \partial_y P + f3(x) P$. The range of x goes from -$\infty$ to xT, and y goes form -$\infty$ to $\infty$

There is an absorbing boundary for x = xT so P(xT,y,t) = 0, but I want to reinsert the flux through this boundary in x=xR. Thus, there will be a discontinuity in the derivative at x=xR: $\partial_x P(x=xR^+,y,t) - \partial_x P(x=xR^-,y,t) = \partial_x P(x=xT,y,t)$, and the function P will be continuous at x=xR.

So I'm thinking in separating the plane in two regions, one for x<xR and another for x>=xR. But I don't know how to write the boundary conditions at the interface x=xR (the two conditions are continuity in x=xR and a jump in the first derivative)

I'd really appreciate if someone can help me with this

**Comment by david-zwicker** [comment]
I'm afraid that such a special case will be not straight-forward to implement in `py-pde`. However, you might be able to write your own class that implements the PDE, calculates the flux across the boundary conditions and re-inserts it as a local source at `x=R`, but I'm not even sure I understand the problem correctly.
